src/WCSimEnumerations.cc

```diff
@@ -100,6 +101,78 @@ std::string WCSimEnumerations::EnumAsString(ReflectionSurface_t r)
   }
   return "";
 }
+ 
+std::string WCSimEnumerations::EnumAsString(ProcessType_t p)
```

Comment:
Please make these strings longer & more descriptive as suggested below
(Don't need to worry about memory/disk space - it's only what's being printed to screen in various places)

Reply:
👍